clang/tools/libclang/CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ if(ENABLE_SHARED)
     set_target_properties(libclang
       PROPERTIES
       VERSION ${LIBCLANG_LIBRARY_VERSION}
-      DEFINE_SYMBOL _CINDEX_LIB_)
+      DEFINE_SYMBOL _CINDEX_LIB_  DEFINE_SYMBOL CLANG_EXPORTS)
```

Comment:
This will override the `_CINDEX_LIB_` symbol, so this seems incorrect.

Reply:
I've fixed this and changed it to CLANG_BUILD_STATIC as well as adding a comment why its needed